src/Http/Controllers/RemoveController.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of Starbs Yeh by Graham Campbell.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+
+namespace Starbs\Yeh\Http\Controllers;
+
+use Starbs\Http\Controllers\AbstractController;
+
+class RemoveController extends AbstractController
+{
+    /**
+     * Do some clever things, then return a response.
+     *
+     * @return \Symfony\Component\HttpFoundation\Response
+     */
+    protected function fire()
+    {
+        $image = $this->container->get('factory')->remove($this->args['id']);
+
+        if (!$image) {
+            return $this->success(['message' => 'Image removed'], 410);
+        }
+
+        if ($image) {
+            return $this->error(['message' => 'Image not removed. An error occured.'], 500);
+        }
+
+        return $this->error(['message' => 'Not Found'], 404);
```

Comment:
There's way more than that. We're using UNSIGNED BIGINTEGER. We've got 18446744073709551615. :)